.eslintrc.json

```diff
@@ -3,6 +3,7 @@
   "overrides": [
     {
       "excludedFiles": [
+        "stake-token-prices/**/*",
```

Comment:
_I think_ if you add the glob in a overrides, you don't need to add them here.

```suggestion
```


Note that the other overridden values are not listed in `excludedFiles`